src/Symfony/Component/Console/Application.php

```diff
@@ -1184,4 +1184,9 @@ private function init()
             $this->add($command);
         }
     }
+
+    private function getPhpVersion(): string
+    {
+        return sprintf('PHP <info>%s</info> (binary: %s)', PHP_VERSION, PHP_BINARY);
```

Comment:
I wouldn't add the path to the binary as that can be quite long.
For the version, this constant contains the extra part, which is noise mostly.
Let's use `PHP_*_VERSION` instead.